.github/workflows/release.yml

```diff
@@ -0,0 +1,39 @@
+name: CDK Deployment
+
+on:
+  push:
+    branches:
+      - main
+      - stg
+      - dev
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v2
+      with:
+        repository: 'AI4Organization/faiss-backend-with-langchain-python-coreservices'
+        ref: ${{ github.ref }}
+        ssh-key: ${{ secrets.SSH_KEY }}
+
+    - name: Set up AWS CLI
+      uses: aws-actions/configure-aws-credentials@v1
```

Comment:
**Suggestion:** Pin the 'aws-actions/configure-aws-credentials' action to a specific version to avoid breaking changes in future releases.
```suggestion
      - name: Set up AWS CLI
        uses: aws-actions/configure-aws-credentials@v1.5.5 # Pin to a specific version for stability
```